src/skmatter/clustering/_quick_shift.py

```diff
@@ -20,19 +20,19 @@ class QuickShift(BaseEstimator):
     Parameters
     ----------
     dist_cutoff_sq : float, default=None
-        The squared distance cutoff for searching for the next point. If `None`, the
-        Gabriel graph is used.
+        The squared distance cutoff for searching for the next point. If :obj:`None`,
+        the Gabriel graph is used.
     gabriel_shell : int, default=None
         The number of neighbor shell of gabriel graph. If None, the distance cutoff
         is used.
     scale : float, default=1.0
         Distance cutoff scaling factor used during the QS clustering. It will be squared
         since we are using the squared distance.
-    metric : str, default='periodic_euclidean'
+    metric : Callable, default=``pairwise_euclidean_distances``
         The metric to use. Currently only one.
     metric_params : dict, default=None
         Additional parameters to be passed to the use of
-        metric.  i.e. the cell dimension for `periodic_euclidean`
+        metric.  i.e. the cell dimension for ``pairwise_euclidean_distances``
```

Comment:
Can you link to the docs page of `pairwise_euclidean_distances `.

With something like  

```
:func:`skmatter.metrics. pairwise_euclidean_distances()`
```

or similar.